<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoiceChatController</class>
 <widget class="QDialog" name="VoiceChatController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="pages">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="hostPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="hostLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="availableVoiceInputDevicesLabel">
           <property name="text">
            <string>Available voice input devices:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="availableVoiceInputDevicesComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="startSessionBtn">
         <property name="text">
          <string>Start session</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="joinPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="joinLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="iPAddressLabel">
           <property name="text">
            <string>IP Address: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="iPAddressLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="portLabel">
           <property name="text">
            <string>Port: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="portLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
